packages/api/src/routes/v3/report.ts

```diff
@@ -28,13 +29,17 @@ export default (server: FastifyInstance, _options: RouteShorthandOptions, done:
             version: {
               type: 'string',
             },
+            explanation: {
+              type: 'string',
+            },
           },
           required: ['url', 'version'],
           type: 'object',
         },
       },
     },
     async (request, reply) => {
+      console.log(request.body);
```

Comment:
Please, can you remove this unnecessary console statement?

Reply:
Done